locale/pl/libgammu.po

```diff
@@ -346,7 +346,7 @@ msgstr "Żądana funkcjonalność została wyłączona podczas kompilacji."
 #: libgammu/gsmcomon.c:146
 #, fuzzy
```

Comment:
As long as the translation is marked fuzzy, it's not used, that was probably not the intention...